boards/arm/stm32h7b3i_dk/stm32h7b3i_dk.dts

```diff
@@ -224,3 +224,44 @@
 	def-back-color-green = <0xFF>;
 	def-back-color-blue = <0xFF>;
 };
+
+&octospi1 {
+	pinctrl-0 = <&octospim_p1_clk_pb2 &octospim_p1_ncs_pg6
+		     &octospim_p1_io0_pd11 &octospim_p1_io1_pf9
+		     &octospim_p1_io2_pf7 &octospim_p1_io3_pf6
+		     &octospim_p1_io4_pc1 &octospim_p1_io5_ph3
+		     &octospim_p1_io6_pg9 &octospim_p1_io7_pd7
+		     &octospim_p1_dqs_pc5>;
+	pinctrl-names = "default";
+
+	status = "okay";
+
+	mx25lm51245: ospi-nor-flash@0 {
+		compatible = "st,stm32-ospi-nor";
+		label = "MX25LM51245";
```

Comment:
Label property are being wiped out all over from the repository, https://github.com/zephyrproject-rtos/zephyr/pull/47881 wiped them from this binding. Can you drop them from here as well?